tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -140,14 +140,21 @@ export const getCardMetadata = createSelector(
   }
 );
 
-// A cheap identity selector to skip recomputing selectors.
-const getVisibleCardIdSet = createSelector(
+// A cheap identity selector to skip recomputing selectors when `state` changes.
+const selectVisibleCardIdSet = createSelector(
   selectMetricsState,
   (state): Set<CardId> => {
     return state.visibleCards;
   }
 );
 
+export const getVisibleCardIdSet = createSelector(
+  selectVisibleCardIdSet,
+  (cardIdSet): Set<CardId> => {
+    return cardIdSet;
+  }
+);
+
```

Comment:
Hm, if we expose the Set, it's probably simpler if we got rid of the Array variant, and avoid keeping 2 selectors that do almost the same thing.  wdyt?